hist/hist/inc/Math/WrappedMultiTF1.h

```diff
@@ -270,16 +302,16 @@ namespace ROOT {
             // case of polynomial function (no parameter dependency)  (case for dim = 1)
             assert(fDim == 1);
             if (ipar == 0) return 1.0;
+#ifdef R__HAS_VECCORE
+            return vecCore::math::Pow(x[0], static_cast<T>(ipar));
```

Comment:
Not if you want to be able to use ROOT without VecCore, unfortunately. That's why we need to do this in a generic way in TMath, as discussed yesterday. Then the code using the math functions can just call the ROOT math function and the right version will be used for both scalar and simd code.